doc/src/manual/documentation.md

```diff
@@ -338,7 +338,7 @@ function f end
 f
 ```
 
-Adds docstring `"..."` to function `f`. The first version is the preferred syntax, however both
+Adds docstring `"..."` to the `Function` `f`. The first version is the preferred syntax, however both
```

Comment:
I don't think the `Function` type is especially relevant here.

Reply:
Don't we want to highlight that though? In contrast with L357 which explains how to add docstring for a specific `Method`?

Reply:
For that reason I think saying `function` is ok, but `f` is not necessarily of type `Function`. It might be a Type, or in the case of documenting just `f` it might be anything.

I didn't notice it before, but referring to the `Method` type below also seems spurious. I would use lowercase "method". I believe you can't even use `Method` objects to add method documentation.